schemas/com.google.privacysandbox/topics/jsonschema/1-0-0

```diff
@@ -0,0 +1,34 @@
+{
+    "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+    "self": {
+        "vendor": "com.google.privacysandbox",
+        "name": "topics",
+        "version": "1-0-0",
+        "format": "jsonschema"
+    },
+    "description": "Privacy Sandbox Topics. You can learn more at https://developer.chrome.com/docs/privacy-sandbox/topics/",
+    "type": "object",
+    "properties": {
+        "topics": {
+            "type": [ "array", "null"],
+            "items": {
+                "description": "Topic entry",
```

Comment:
I think you need `"type": "object"` somewhere around here, inside the `items` block.

Currently, the following JSON would be technically valid against this schema:

```json
{
  "schema": "iglu:com.google.privacysandbox/topics/jsonschema/1-0-0",
  "data": {
    "topics": [
      "this", "is", "not", "what", "you", "want!"
    ]
  }
}
```

You could also add  `additionalProperties: false` inside the `items` block.

Reply:
Yeap, thanks for the catch! Fixed and validated on micro